src/aui-form-builder/tests/unit/js/aui-form-builder-field-base-tests.js

```diff
@@ -319,11 +319,15 @@ YUI.add('aui-form-builder-field-base-tests', function(Y) {
 
             Y.Assert.isTrue(Y.one('.form-builder-field-settings-panel-advanced-content').hasClass('toggler-content-expanded'));
 
-            Y.one('.form-builder-field-settings-panel-advanced-button').simulate('click');
-            Y.Assert.isTrue(Y.one('.form-builder-field-settings-panel-advanced-content').hasClass('toggler-content-collapsed'));
+            Y.one('.form-builder-field-settings-panel-toggler-advanced').simulateGesture('tap');
```

Comment:
Can you create a new test to this `tap` event instead of change the event `click` to `tap`?